canonical-web-and-design/snapcraft.io
# Add arabic language to publicise

## Done

[List of work items including drive-bys]

## Issue / Card

Fixes #2998 

## QA

- Pull the branch
- Run the site using the dotrun snap with `$ dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8004/<snap_name>/publicise
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Select arabic language and see everything makes sense

## Screenshots

![image](https://user-images.githubusercontent.com/40214246/89547107-4276c880-d7fd-11ea-8d3d-51a50f3a5ad0.png)
